tests/epilogue.test.js

```diff
@@ -34,6 +34,22 @@ describe('Epilogue', function() {
     done();
   });
 
+  it('should extend .resource options if they are provided on initialize', function() {
+    var resourceOptions = { pagination: false };
+
+    epilogue.initialize({
+      app: {},
+      sequelize: new Sequelize('main', null, null, {
+        dialect: 'sqlite',
+        storage: ':memory:',
+        logging: (process.env.SEQ_LOG ? console.log : false)
+      }),
+      resource: resourceOptions
```

Comment:
I think `{ defaults: { resource ...` is a good compromise. The existing keys are going to be global to the module (`app`, `sequelize`). Arguably, so would have been `resource`, as after all epilogue is chiefly interested in creating resources - however, I get your point and I think this would be a good middle ground.